youtube/views.py

```diff
@@ -8,16 +8,21 @@
 from .api import youtube_search
 
 YOUTUBE_RES_TYPES = ('video', 'channel', 'playlist')
-
+YOUTUBE_MAX_RESULTS = 50
+DEFAULT_MAX_RESULTS = 25
 
 def search(request):
     search_term = request.GET.get('search_term', '')
     restype = request.GET.get('restype', 'video')
-    max_results = request.GET.get('max_results', 25)
-    if max_results > 50:
-        max_results = 50
-    if max_results < 0:
-        max_results = 0
+    max_results = request.GET.get('max_results', DEFAULT_MAX_RESULTS)
+
+    try:
+        max_results = int(max_results)
+    except ValueError, TypeError:
```

Comment:
You should add parenthesis around these two exceptions. The way it's written now will change the value of TypeError class to an instance of ValueError the first time ValueError is thrown. See http://www.toptal.com/python/top-10-mistakes-that-python-programmers-make#common-mistake-3-specifying-parameters-incorrectly-for-an-exception-block